README.md

```diff
@@ -14,24 +14,54 @@ Add/write sourcemaps to/from Vinyl files.
 
 ```js
 sourcemap.add(file, function(err, updatedFile) {
-
+  // updatedFile will have a .sourceMap property
 });
 
+// The 2nd argument can be given as a path string
 sourcemap.write(file, './maps', function(err, updatedFile, sourcemapFile) {
+  // sourcemapFile will be a Vinyl file to be written to some location
```

Comment:
Thanks for taking a look.  Our standard gulp-org docs template has the simplified usage first followed by the detailed API docs, so I'll probably keep it in this order.